package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "gulp test && gulp prod",
     "coverage": "gulp coverage",
+    "release": "./release.sh",
```

Comment:
OOH nvm, i see the file now, i don't know why, but at first i thought it was a `README` update.